pkgs/applications/editors/emacs/elisp-packages/manual-packages/beancount/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, melpaBuild
+, fetchFromGitHub
+, emacs
+, writeText
+,
+}:
+let
+  rev = "519bfd868f206ed2fc538a57cdb631c4fec3c93e";
```

Comment:
I don't see why you put `rev` in the let block but nothing else.

Reply:
`rev` is used in `src` and `commit`.

Reply:
I get that. What I was wondering why `sha256` & others that will be updated in lock-step are not also in the `let` block.

Reply:
Putting only `rev` in `let` seems to be a common pattern in manual packages.

> What I was wondering why sha256 & others that will be updated in lock-step are not also in the let block

May I ask what is the advantage of doing so?